eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -2125,11 +2133,12 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
   private def handleLocalError(cause: Throwable, d: Data, msg: Option[Any]) = {
     cause match {
       case _: ForcedLocalCommit => log.warning(s"force-closing channel at user request")
+      case _ if stateName == WAIT_FOR_OPEN_CHANNEL => log.info(s"${cause.getMessage} while processing msg=${msg.getOrElse("n/a").getClass.getSimpleName} in state=$stateName")
       case _ => log.error(s"${cause.getMessage} while processing msg=${msg.getOrElse("n/a").getClass.getSimpleName} in state=$stateName")
     }
     cause match {
       case _: ChannelException => ()
-      case _ => log.error(cause, s"msg=${msg.getOrElse("n/a")} stateData=$stateData ")
+      case _ => log.error(cause, s"msg=${msg.getOrElse("n/a")} in state=$stateName ")
```

Comment:
Did you see that happening a lot? Everyting should be a `ChannelException`, we may want more details in case of unexpected errors?

Reply:
No I didn't see that one happen, but I looked for places where we logged the whole `stateData` and removed them. But you're right, since this is something that should never happen it makes sense to log all the state to help troubleshoot.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1817/commits/ee9a1e03f284e3e2250b086cffe369d3d5751017